doc/overview.rst

```diff
@@ -61,9 +61,9 @@ Terminology
    `dune-workspace` file.
 
 -  **project**: a collection of source files that must include a `dune-project`
-   file. It may also contain one or more packages. Each directory in the tree,
-   including the root, must have a `dune` file specifying how to build the files
-   in its directory. Projects can be shared between different applications.
+   file. It may also contain one or more packages. Instructions to build files
+   in a directory are contained in a `dune` file in this directory. Projects can
```

Comment:
I think we should say directory or subdirectory.

Reply:
I'm not sure what the suggestion is (the sentence contains "directory" already). What I'd like to say is: Instructions to build files in directory `a/` go in `a/dune`.

Reply:
I'm suggesting that we mention dune files may appear in subdirectories otherwise it might look like only the currently directory can have them. "Instructions to build files in a directory of subdirectory...".